src/config/config.cc

```diff
@@ -755,6 +756,24 @@ void Config::initFieldCallback() {
       {"tls-session-cache-size", set_tls_option},
       {"tls-session-cache-timeout", set_tls_option},
 #endif
+      {"histogram-bucket-boundaries",
+       [this]([[maybe_unused]] Server *srv, [[maybe_unused]] const std::string &k, const std::string &v) -> Status {
+         std::vector<std::string> buckets = util::Split(v, ",");
+         histogram_bucket_boundaries.clear();
+         if (buckets.size() < 1) {
+           return Status::OK();
+         }
+         std::transform(buckets.begin(), buckets.end(), std::back_inserter(histogram_bucket_boundaries),
+                        [](const std::string &val) { return std::stod(val); });
```

Comment:
`stod` will throw exceptions while it fails to parse the string to number. We can use `ParseInt` in `parse_util.cc`.